pywbem/exceptions.py

```diff
@@ -78,10 +122,18 @@ def __init__(self, status, reason, cimerror=None, cimdetails=None):
             * Value: header field value (i.e. text message)
 
             Passing `None` will result in an empty dictionary.
+
+          conn (:class:`~pywbem.WBEMConnection`): Connection in whose context
+            the error happened. `None` if the error did not happen in context
+            of any connection, or if the connection context was not known.
+
+        :ivar args: A tuple (status, reason, cimerror, cimdetails) set from the
```

Comment:
I noticed that the args doc moved moved up to HTTPError at the same time you inserted the super... However, I do not see where above this in the hierarchy the args actually gets set.  I may be just missing it.

Reply:
The args doc on HTTPError was missing, and adding it is a docs fix that is independent of all the other changes.

The args instance variable is set in the Python Exception class, and the previous implementation duplicated that into HTTPError (and into CIMError, fwiw). Now that our Error class has an init method, we can use the more logical approach of delegating that into the Python Exception class. Maybe that would have worked even without an init method in the Error class in between, but I never tried. In any case, the unit tests verify that the args instance variable is set.